### ISSUE TITLE
cairocffi requirements.txt giving VersionConflict error

### ISSUE THREAD

**Issue by innomation** [graphite-project/graphite-web#2095]
I've built a script to install graphite-web (and the rest of the graphite/grafana stack) which has worked fine without any issues for the past year until today. It was also working earlier this week. 

I am now getting a VersionConflict error when installing the cairocffi package listed in graphite-web's pip requirements.txt file. This 'bug' has just been caught today, and I'm trying to figure out if there were any changes made that would cause this? The requirements does not include a sticky version, and it keeps trying to pull v0.8.6, but requires VersionConflict requires >= 1.1.0. 

<img width="1011" alt="screen shot 2017-11-01 at 2 36 22 pm" src="https://user-images.githubusercontent.com/10753462/32299368-9f151bb6-bf12-11e7-8609-c97a77b4b1de.png">

Anyone able to shed some light onto this? I've tried using graphite-web 0.9.15 (used in the original script), and 1.0.2, same problem. 

Additional Details:
Centos 7.2.1511 (fresh ISO install)
Python 2.7.5
Django v1.9.13

Install Graphite-Web BASH Function
       # declare graphite version
        # version=0.9.15
        version="1.02" 
       # install graphite-web
        echo "----------------------------"
        echo " Installing Graphite-Web..."
        echo "----------------------------"
        # cffi VersionConflict bug fix
        # pip install --upgrade pip; pip install --upgrade cffi
        sudo git clone https://github.com/graphite-project/graphite-web.git /usr/local/src/graphite-
web
        cd /usr/local/src/graphite-web && sudo git checkout ${version}
        **cd /usr/local/src/graphite-web && sudo -E pip install -r requirements.txt**
        cd /usr/local/src/graphite-web && sudo -E pip install rrdtool python-ldap
        cd /usr/local/src/graphite-web && python check-dependencies.py
        cd /usr/local/src/graphite-web && sudo -E python setup.py install

**Comment by innomation** [comment]
A 'yum -y update' installed the newest version of python-cffi, which brought it up to v1.6, and resolved the issue. CentOS can't use PIP to fulfill the cairocffi version requirement. I hope this helps someone in the future.

**Comment by DanCech** [comment]
Thanks for the info, seems like we should pin the cairocffi requirement

**Comment by innomation** [comment]
I've tried pinning it manually and running, but had no luck. However, it wouldn't hurt to replicate the tests to confirm. If confirmed, my guess is older version installed and managed by yum package manager. 

Ran a few tests, a 'yum update -y' will also update the packages required for scripts to pass. I jumped the gun too quickly and ran install on fresh 7.2 ISO out of the box in a dev environment.
